src/components/ProductQuantity/index.js

```diff
@@ -17,13 +17,14 @@ function ProductQuantity({ price }) {
   return (
     <div>
       <label htmlFor="quantity">Quantity:</label>
-      <input type="number" id="quantity" name="quantity" min="1" value={quantity} />
-      <button type="button" onClick={incrementQuantity}>
-        +
-      </button>
       <button type="button" onClick={decrementQuantity}>
         -
       </button>
+      <input style={{ width: '50px', textAlign: 'center' }} maxLength={5} type="number" id="quantity" name="quantity" min="1" value={quantity} />
```

Comment:
`label` and `input` elements should only be used inside of a `form` element; I'm not sure what's happening here, can you explain their usage?